x-pack/plugins/fleet/public/applications/fleet/sections/settings/components/legacy_settings_form/index.tsx

```diff
@@ -9,16 +9,12 @@ import React, { useEffect, useCallback } from 'react';
 import styled from 'styled-components';
```

Comment:
This file is just the old settings flyout, that will be removed in coming PRs for multiple output UI